### PR TITLE
Updated the population Check it!

### PR DESCRIPTION
**Pull request by JamesDunton**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -253,6 +253,54 @@ static void CityMenu(Statistics cityStats)
...
+                        } while (!valid);
+                        Console.WriteLine("\nWhich city do you want to change the population?\n");
+                        city = GetCityChoice(cityStats);
+                        string oldPopulation = string.Format("{0:n0}", city.GetPopulation());
+                        Console.WriteLine($"\nThe city of{city.GetCityName()}, {city.GetProvince()} has a current population of {oldPopulation}.");
+                        do
+                        {
+                            Console.Write($"Please enter the new population for {city.GetCityName()}, {city.GetProvince()} :");
+                            isULong = UInt64.TryParse(Console.ReadLine(), out newPopulation);
+                            if (!isULong)
```
Comment (conninator2000) [inline Program.cs:296]: This can be condensed into 1 line

Comment (conninator2000) [reply Program.cs:296]: with an else{ break;} (and make the do-while(true))

File: Program.cs [hunk 2]
```diff
@@ -253,6 +253,54 @@ static void CityMenu(Statistics cityStats)
...
+                        Console.WriteLine("Available data files:\n");
+                        Console.WriteLine("\t1) Canadacities-JSON.json");
+                        Console.WriteLine("\t2) Canadacities-XML.xml");
+                        Console.WriteLine("\t3) Canadacities.csv");
+                        Console.Write("Please make a selection(ex. 1, 2): ");
+                        do
+                        {
+                            selection = Console.ReadLine();
+                            switch (selection.ToLower())
+                            {
```
Comment (hunter-bennett28) [inline Program.cs:270]: you don't need ToLower on digits

**Review by hunter-bennett28 — APPROVED** [review]

File: Program.cs [hunk 3]
```diff
@@ -253,6 +253,54 @@ static void CityMenu(Statistics cityStats)
...
+                        Console.WriteLine($"\nThe city of{city.GetCityName()}, {city.GetProvince()} has a current population of {oldPopulation}.");
+                        do
+                        {
+                            Console.Write($"Please enter the new population for {city.GetCityName()}, {city.GetProvince()} :");
+                            isULong = UInt64.TryParse(Console.ReadLine(), out newPopulation);
+                            if (!isULong)
+                                Console.Write("Invalid response. ");
+                        }
+                        while (!isULong);
+                        cityStats.UpdatePopulation($"{city.GetCityName()}|{city.GetProvince()}", newPopulation, fileName);
```
Comment (conninator2000) [inline Program.cs:300]: Make sure to put this in a try-catch, the method throws exceptions

Comment (conninator2000) [reply Program.cs:300]: Also take a look at the UpdatePopulation method, it accesses cities by the old way, we need to update it to use getSpecificCity

Comment (conninator2000) [reply Program.cs:300]: Bonus points of you comment that method comment for it

**Review by conninator2000 — CHANGES_REQUESTED** [review]
Good, needs a few tweaks then we are good to put it in and get onto the next (and hopefully last) wave of testing

**Pull request closed without merging**